### PR TITLE
Add doc about Submit to Evaluation widget

### PR DESCRIPTION
**Pull request by thomasyu888**
Add instructions on how to use Submit to Evaluation widget. Note that:
* only way to submit a project
* not able to submit docker this way

### CODE REVIEW

File: articles/evaluation_queues.md [hunk 1]
```diff
@@ -185,3 +185,16 @@ Clicking **Refresh Columns** will add these default columns.
...
+
+### Adding Submit to Evaluation Widget
+
+<img style="width: 80%;" src="/assets/images/add_submission_widget.png">
+
+Aside from directing people to submit to an evaluation queue by navigating to an Synapse Entity, you can also add a `Submit To Evaluation` widget on a wiki page. Here you will get an option of allowing people to submit to evaluation queues of a project or a single evaluation queue.
+
+<img style="width: 80%;" src="/assets/images/submit_to_evaluation_widget.png">
+
+**Alpha Mode Feature:** Present a custom HTML form to the user to fill out during submission.  This form is built using [react JSON schema](https://react-jsonschema-form.readthedocs.io/en/latest/). Both a `ui` and `JSON schema` need to be uploaded into Synapse.  A Parent Container must also be assigned when using this feature.  This Container must have Synapse Folders named after the Synapse userids that may be using this evaluation queue service.  These users must have `write` permissions to their respective directories.
```
Comment (thomasyu888) [inline articles/evaluation_queues.md]:
```suggestion
```

File: articles/evaluation_queues.md [hunk 2]
```diff
@@ -185,3 +185,15 @@ Clicking **Refresh Columns** will add these default columns.
...
+### Adding Submit to Evaluation Widget
+
+<img style="width: 80%;" src="/assets/images/add_submission_widget.png">
+
+Aside from directing people to submit to an evaluation queue by navigating to an Synapse Entity, you can also add a `Submit To Evaluation` widget on a wiki page. Here you will get an option of allowing people to submit to evaluation queues of a project or a single evaluation queue.
+
+<img style="width: 80%;" src="/assets/images/submit_to_evaluation_widget.png">
+
+
+<img style="width: 80%;" src="/assets/images/submission_react_widget.png">
```
Comment (thomasyu888) [inline articles/evaluation_queues.md]: ```suggestion```

**Comment by kelshmo** [comment]
@thomasyu888 What further actions should be taken with this PR? Thanks!

**Comment by thomasyu888** [comment]
Thanks @kelshmo, this is all.

File: _articles/evaluation_queues.md [hunk 3]
```diff
@@ -214,6 +214,16 @@ If you are happy with your leaderboard configurations, save both the configurati
 
 <img style="width: 80%;" src="/assets/images/leaderboard_on_wiki.png">
 
+
+### Adding Submit to Evaluation Widget
+
+<img style="width: 80%;" src="/assets/images/add_submission_widget.png">
+
+Aside from directing people to submit to an evaluation queue by navigating to an Synapse Entity, you can also add a `Submit To Evaluation` widget on a wiki page. Here you will get an option of allowing people to submit to evaluation queues of a project or a single evaluation queue.  Currently this is the only way you can submit Synapse `Projects`, but you cannot submit Synapse Docker repositories.
```
Comment (kelshmo) [inline _articles/evaluation_queues.md:222]:
```suggestion
You may embed a `Submit To Evaluation` widget on a Wiki page to improve visibility of your Evaluation queue. The widget allows participants to submit to multiple Evaluation queues within a Project or a single Evaluation queue. 

Currently, this Wiki widget is required to submit Synapse Projects to an Evaluation queue. Synapse Docker repositories can not be submitted through this widget.
```

Comment (kelshmo) [reply _articles/evaluation_queues.md:222]: Mostly style updates.

Do my edits preserve the meaning of your intended edits?

**Review by kelshmo — CHANGES_REQUESTED** [review]
I'm not so familiar with Evaluation queues, so let me know if these proposed edits change the meaning. Thanks Tom!

File: _articles/evaluation_queues.md [hunk 4]
```diff
@@ -214,6 +214,18 @@ If you are happy with your leaderboard configurations, save both the configurati
 
 <img style="width: 80%;" src="/assets/images/leaderboard_on_wiki.png">
 
+
+### Adding Submit to Evaluation Widget
```
Comment (thomasyu888) [inline _articles/evaluation_queues.md:218]:
```suggestion
### Submit to an Evaluation Queue from a Wiki Page
```

File: _articles/evaluation_queues.md [hunk 5]
```diff
@@ -214,6 +214,16 @@ If you are happy with your leaderboard configurations, save both the configurati
...
+
+<img style="width: 80%;" src="/assets/images/add_submission_widget.png">
+
+Aside from directing people to submit to an evaluation queue by navigating to an Synapse Entity, you can also add a `Submit To Evaluation` widget on a wiki page. Here you will get an option of allowing people to submit to evaluation queues of a project or a single evaluation queue.  Currently this is the only way you can submit Synapse `Projects`, but you cannot submit Synapse Docker repositories.
+
+<img style="width: 80%;" src="/assets/images/submit_to_evaluation_widget.png">
```
[the remark this answers is not in the document]
Comment (thomasyu888) [reply _articles/evaluation_queues.md:226]: Perhaps,  I _think_ these are some of the scenarios where a evaluation queue would be unavailable:

1. User does not have permission to view the evaluation queues
2. The specified challenge doesn't have evaluation queues. 
3. There are no "open" evaluation queues (you can set quotas on time durations on evaluation queues and have them automatically 'close' after some time)

Both of these messages would occur after the creation of his widget though.

[the remark this answers is not in the document]
Comment (kelshmo) [reply _articles/evaluation_queues.md:226]: In the image you uploaded of the widget - there is a field to enter an "Evaluation Queue unavailable message" and I'm not sure that it is clear that the creator can, or why they would, customize this message. 

The points you listed here might be relevant to be incorporate into "Configuring Leaderboard Widget"

**Review by kelshmo — COMMENTED** [review]
Should we incorporate those points about EQ unavailability into the "Configuring Leaderboard Widget" section?

File: _articles/evaluation_queues.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (kelshmo) [reply _articles/evaluation_queues.md:226]: See #755 and #754

**Review by kelshmo — APPROVED** [review]

**Pull request merged**